lib/src/Uncertainty/StatTests/LinearModelTest.cxx

```diff
@@ -47,7 +47,7 @@ LinearModelTest::LinearModelTest()
 /*  */
 TestResult LinearModelTest::LinearModelFisher(const Sample & firstSample,
     const Sample & secondSample,
-    const LinearModel & linearModel,
+    const Point & trendCoefficients,
     const Scalar level)
```

Comment:
why relying on coefficients only ?
We might rely on `LinearModelResult`

Reply:
ok